js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "walletlink",
-  "version": "2.4.6",
+  "version": "2.4.7",
```

Comment:
Do you need to update the `version.ts` for the new version

Reply:
version.ts is auto-generated when `yarn build:prod` is run

Reply:
ahh thats right, its part of the build step!